make/run

```diff
@@ -33,15 +33,18 @@ kubectl get storageclass persistent 2>/dev/null || {
 }
 
 source bin/settings/settings.env
-source bin/settings/network.env
+source <(sed -e '/^DOMAIN=/d' -e '/^UAA_HOST=/d' bin/settings/network.env)
+DOMAIN="${DOMAIN:-$(grep '^DOMAIN=' bin/settings/network.env | sed 's/DOMAIN=//')}"
```

Comment:
I would propose to go with how Jan is doing it, i.e. add cd ${GIT_ROOT} at the beginning and then remove all the other GIT_ROOT refs.